src/test/java/test/github1490/FactoryMethodSample.java

```diff
@@ -0,0 +1,33 @@
+package test.github1490;
```

Comment:
Maybe `test.factory.github1490`?

Reply:
The package `github1490` already has some Factory related classes. Are you suggesting that I move all of them into the new package `test.factory.github1490` ? Since logically speaking GITHUB1490  and GITHUB1515 together constitute the listener functionality, I thought of housing this class also in the same package.

Reply:
Each time I have a like at tests, I dislike github* package because I don't understand easily what feature(s) they test.
I think it helps if all tests are located under a feature package.
I don't know yet what should be the package name of the test which covers more than 1 feature ;)